website/content/en/build/_index.html

```diff
@@ -9,11 +9,11 @@
 ---
 
 <section class="of-masthead of-masthead--subpage">
-  <h1 class="of-heading of-heading--xl">Building the Operator SDK</h1>
+  <h1 class="of-heading of-heading--xl">Building Operators using the Operator SDK</h1>
 </section>
 <section class="of-section-page-intro of-section-page-intro--overlay">
   <header class="of-section-page-intro__header">
-    <h2 class="of-heading of-heading--md">How can I build with Operator SDK?</h2>
+    <h2 class="of-heading of-heading--md">How can I write an Operator with Operator SDK?</h2>
   <p>There are many ways to build an Operator with SDK. To get started building today, follow the steps in this quick start guide.</p>
```

Comment:
May would be better here be a link for the Getting started .. however, I think we have not this doc yet. So, 👍